### PR TITLE
fix label to be RFC compliant

### PR DESCRIPTION
**Pull request by jonasheinisch**
### Related Issues
- #9835

### Checklist
- [ ] e2e cypress tests locally verified.
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on final implementation.
- [ ] Documentation provided. (Add links if there's any)
- [ ] Unit tests provided. (Add links if there's any)
- [ ] Integration tests provided. (Add links if there's any)

### Related PRs
- #1268 

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: modules/i18n/src/translations/en-US/portals/console.ts [hunk 1]
```diff
@@ -1670,7 +1670,7 @@ export const console: ConsoleNS = {
                                         hint: "Select to issue a new <1>refresh_token</1> each time a "+
                                             "<3>refresh_token</3> is " +
                                             "exchanged. The existing token will be invalidated.",
-                                        label: "Rotate refresh token",
+                                        label: "Renew refresh token",
```
Comment (brionmario) [inline modules/i18n/src/translations/en-US/portals/console.ts:1673]: cc:/ @janakamarasena @mefarazath

**Comment by brionmario** [comment]
> [CLA assistant check](https://cla-assistant.io/wso2/identity-apps?pullRequest=2787) Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/wso2/identity-apps?pullRequest=2787) before we can accept your contribution.You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/wso2/identity-apps?pullRequest=2787) it.

@jonasheinisch 
Thanks a lot for the contribution. Could you please sign the CLA?

**Comment by jonasheinisch** [comment]
Done

**Comment by jonasheinisch** [comment]
@brionmario Anything else needed?

**Comment by NipuniBhagya** [comment]
Hi @jonasheinisch, 

We really appreciate your contribution and are sorry about the delay in merging it. While we agree with your suggestion we took some time to evaluate the impact of this change on our other solutions. We are looking forward to more contributions from you in the future.

Thanks,
Nipuni.

**Review by NipuniBhagya — APPROVED** [review]

**Pull request merged**